source/_integrations/workday.markdown

```diff
@@ -99,9 +99,6 @@ binary_sensor:
     country: US
     workdays: [mon, tue, wed, thu, fri]
     excludes: [sat, sun]
-    add_holidays:
-      - "2020-02-24"
-      - "2020-04-25"
 ```
```

Comment:
The suggested change has been closed as a bug before. I'm not sure what changed?
Originally, this worked. one could add custom holidays, without including the full holiday calendar for your country.

Reply:
Sorry, coming back to your original question: After looking at the code history, I would say that this never worked (at least since the custom holidays were added in 2019).